.gitignore

```diff
@@ -60,3 +60,6 @@ ios/example-tvOSTests/
 ios/example.xcodeproj/xcshareddata/xcschemes/example-tvOS.xcscheme
 ios/example/Images.xcassets/Contents.json
 ios/exampleTests/
+
+# Keep Expo plugin build
+!plugin/build
```

Comment:
Ideally we don't want to keep generated code in the source and add it as a part of build step